core/services/chainlink/relayer_factory.go

```diff
@@ -301,3 +301,65 @@ func (r *RelayerFactory) NewCosmos(config CosmosFactoryConfig) (map[types.RelayI
 	}
 	return relayers, nil
 }
+
+type AptosFactoryConfig struct {
+	Keystore    keystore.Aptos
+	TOMLConfigs RawConfigs
+}
+
+func (r *RelayerFactory) NewAptos(ks keystore.Aptos, chainCfgs RawConfigs) (map[types.RelayID]loop.Relayer, error) {
+	plugin := env.NewPlugin("aptos")
+	loopKs := &keystore.AptosLooppSigner{Aptos: ks}
+	return r.NewLOOPRelayer("Aptos", corerelay.NetworkAptos, plugin, loopKs, chainCfgs)
+}
+
+func (r *RelayerFactory) NewLOOPRelayer(name string, network string, plugin env.Plugin, ks coretypes.Keystore, chainCfgs RawConfigs) (map[types.RelayID]loop.Relayer, error) {
```

Comment:
nit: consider grouping the keystore, config, and capRegistry in a type like PluginContext